include/mbgl/gfx/context.hpp

```diff
@@ -51,10 +52,13 @@ using UniqueDrawableBuilder = std::unique_ptr<DrawableBuilder>;
 using VertexAttributeArrayPtr = std::shared_ptr<VertexAttributeArray>;
 #endif
 
+static ContextObserver nullObserver;
```

Comment:
This can be `const`. Consider using an anonymous namespace instead.

Reply:
It can be moved to an anonymous namespace but can't be const as it's used to default-initialize a non-const pointer

Reply:
Alright!